Gems/Atom/RHI/DX12/CMakeLists.txt

```diff
@@ -13,3 +13,5 @@ o3de_restricted_path(${gem_json} gem_restricted_path gem_parent_relative_path)
 ly_add_external_target_path(${CMAKE_CURRENT_LIST_DIR}/3rdParty)
 
 add_subdirectory(Code)
+
+ly_create_alias(NAME Atom_RHI_DX12.HeadlessServers NAMESPACE Gem)
```

Comment:
This alias should be added next to the other aliases in the Code/CMakeLists.txt file for the Atom_DX12 Gem.